frontend/src/index.css

```
@@ -0,0 +1,20 @@
+    @import "~@blueprintjs/core/lib/css/blueprint.css";
+    @import "~@blueprintjs/icons/lib/css/blueprint-icons.css";
+}
+
+html, body, #root {
+    height: 100%;
+    padding: 0;
+    margin: 0;
+}
+
+body {
+    font-family: 'Century Gothic', CenturyGothic, AppleGothic, sans-serif;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: CenturyGothic

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)